backend/python/app/services/implementations/comment_service.py

```diff
@@ -172,3 +191,17 @@ def update_comments(self, updated_comments):
 
         # TODO: return updated comments as list of CommentResponseDTO's
         return updated_comments
+
+    def _update_story_translation_last_activity(self, story_translation, is_translator):
```

Comment:
hmmmm I dont like that we have to maintain two copies of this function. could you delete this function in comment_service and then call the function from the story_service? You might need to make an instance of the story service to use this function. see `middlewares/auth.py` as example. it makes a User service and Auth service instance. I havent tried this before so lmk if this doesnt work